art-Creating_a_Service_for_VM_Provisioning/topics/Overview.adoc

```diff
@@ -0,0 +1,12 @@
+[[example-creating-a-service-overview]]
+== Overview
+
+{product-title} provides the ability to define services, which can include one or more virtual machines or instances, and can be deployed across hybrid environments. Services can be made available to users through the Self Service user interface that allows users to order predefined IT services without IT operations getting involved.
```

Comment:
Suggested edit: "...through the Self Service user interface, allowing users to order..."